apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -5,15 +5,45 @@
     <link href="<%= node_modules_asset_path('tablesorter/dist/css/theme.bootstrap.min.css') %>" rel="stylesheet" />
     <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.min.js') %>"></script>
     <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.widgets.min.js') %>"></script>
+    <%- compiled_script_tag('instructorAssessmentInstanceClient.ts') %>
   </head>
   <body>
     <script>
-      $(function () {
-          $('[data-toggle="popover"]').popover({sanitize: false})
+      $(() => {
+        $('[data-toggle="popover"]').popover({ sanitize: false });
       });
     </script>
     <%- include('../partials/navbar', {navPage: ''}); %>
+    <% var assessmentInstanceOwner = assessment.group_work ? instance_group.name : instance_user.name; %>
     <main id="content" class="container-fluid">
+      <div class="modal fade" id="resetQuestionVariantsModal" tabindex="-1" role="dialog" aria-labelledby="resetQuestionVariantsModalLabel">
```

Comment:
Not that it really matters, but in PT we've been putting modals in their own functions so we can include them with `${ResetQuestionVariantsModal(...)}`. I find this helps to keep the page structure clear (we've also been doing it for cards) and I like having each "component" of the page separate. We've also been using the `Modal()` component, but that might not be in PL yet.

Reply:
If this were _not_ an EJS page, I would have positively leapt at the opportunity to not have to write all the moral boilerplate by hand. Alas...